packages/jest-did-matcher/src/matchers/toBeMediaType/predicate.js

```diff
@@ -0,0 +1,5 @@
+export default expected => {
+    const regex = /^\w+\/[-+.\w]+/
```

Comment:
minor nit that we might prefer this to be something like `application/did+...`...a

Reply:
Agree. Do we have the complete list?

Reply:
> minor nit that we might prefer this to be something like `application/did+...`...

I think I disagree, because there are statements in the spec that say something must be a media type, and this is what the predicate is meant for.

Additional tests (such as whether a media type is for a conformant representation of the DID document data model) happens elsewhere.